t/copy_input_fastq.t

```diff
@@ -0,0 +1,40 @@
+#!/bin/usr/env perl
+use warnings;
+use strict;
+
+use lib $FindBin::Bin.'../lib';
```

Comment:
You will need to add `use FindBin;` above this line and then add another `/` in front of the path, so it's like `/../lib`.